src/Workleap.Extensions.OpenAPI/RequiredType/SetPropertyAsRequiredTypeFilter.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Workleap.Extensions.OpenAPI.RequiredType;
+
+internal sealed class SetPropertyAsRequiredTypeFilter : ISchemaFilter
```

Comment:
Just by the name it seems we set every property as Required maybe something more explicit like

ExtractRequiredAttributeFromNullableType

Reply:
Could also link to official documentation that state that nullable and required are mutually exclusive in OpenAPI